scripts/seastar-addr2line

```diff
@@ -99,13 +99,14 @@ class BacktraceResolver(object):
                 addresses = []
                 for obj in m.group(2).split():
                     m = re.match(self.address_re, obj)
+                    #print(f"  >>> '{obj}': address {m.groups()}")
```

Comment:
Normally I'd advise to use a logger with trace/debug level here, but I guess it is an overkill for a tool scrip like this.